deepchem/models/tests/test_mat.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+import tempfile
+import numpy as np
+import deepchem as dc
+from deepchem.feat import MATFeaturizer
+from deepchem.models.torch_models import MATModel
```

Comment:
Move this import within `test_mat_regression` and `test_mat_reload` to avoid the pytest error

Reply:
I've updated it now :+1: